gravitee-apim-portal-webui-next/src/app/api-details/api-tab-subscriptions/api-tab-subscriptions.component.scss

```diff
@@ -20,3 +20,17 @@
   justify-content: center;
   padding: 20px 0;
 }
+
+.api-tab-subscriptions__table {
+  overflow: hidden;
+  border: 1px solid #e6e0ec;
+  border-radius: 16px;
+
+  tr.mat-mdc-row.mdc-data-table__row {
```

Comment:
Avoid using classes specific to material design. They can change names which will create style regressions in the future.

Reply:
Done